src/controllers/auth.ts

```diff
@@ -128,40 +146,59 @@ export const createAccessToken = (email: string) => {
     }
 }
 
-
 export const createRefreshToken = (email: string, tokenVersion: number) => {
-    if(process.env.REFRESH_TOKEN_SECRET){
+    if (process.env.REFRESH_TOKEN_SECRET) {
         const refreshTokenSecret: string = process.env.REFRESH_TOKEN_SECRET
         return sign({ email: email, tokenVersion: tokenVersion }, refreshTokenSecret, { expiresIn: '168h' })
     }
 }
 
-export const revokeRefreshToken = (req: Request, res: Response) => {
-    // Todo read the tokenVersion and email from the refreshToken
-    // Tests will have to be updated accordingly
-    console.log(req.cookies.jid)
-    let newTokenVersion: number = req.body.tokenVersion
-    let email: string = req.body.email
-    if (email == null) {
-        return res.status(401).json({ message: "Missing Email" })
-    }
-    if (newTokenVersion == null) {
-        return res.status(401).json({ message: "Missing Token Version" })
-    }
-    newTokenVersion += 1
-    try {
-        let updatedUser = UserModel.updateOne(
-            { email: email },
-            { tokenVersion: newTokenVersion }
-        )
-        return res.status(200).json({
-            message: "Revoked refresh_token successfully"
-        })
-    }
-    catch (err) {
-        console.log(err)
-        return res.status(500).json({
-            message: "Problem with revoking the token"
-        })
+export const revokeRefreshToken = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I can't see how next: NextFunction is used in this function, is it then really needed?